### PR TITLE
Remove binaries and add .gitingore

### PR DESCRIPTION
**Pull request by ismilak**
Removing .o and executable files from the repository and set them untrackable in .gitignore

### CODE REVIEW

**Review by manojnacsl — APPROVED** [review]
Looks good. Approving.

File: .gitignore
Comment (Wi-SUN) [inline .gitignore:1]: Looks OK

**Review by ghost — APPROVED** [review]

**Review by ghost — APPROVED** [review]

**Review by ghost — APPROVED** [review]

**Comment by ismilak** [comment]
> Looks good. Approving.

Hi! Thank you. When will it be merged?

**Comment by manojnacsl** [comment]
> > Looks good. Approving.
> 
> Hi! Thank you. When will it be merged?

You could now use "Squash and merge" in the PR below to merge the approved changes.

**Comment by ismilak** [comment]
> > > Looks good. Approving.
> > 
> > 
> > Hi! Thank you. When will it be merged?
> 
> You could now use "Squash and merge" in the PR below to merge the approved changes.

I think I don't have write permission for the repo, therefore I can't merge it

**Review by paduffy — APPROVED** [review]

**Pull request merged**